### PR TITLE
Add readme.txt file to be included in the NuGet

### PR DESCRIPTION
**Pull request by elamalani**
Add readme.txt file to be included in the NuGet package which opens up after any public App Center Nuget is installed in an application.

### CODE REVIEW

File: nuget/readme.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+-------------------------------------------
+App Center SDK for .NET Readme
+-------------------------------------------
+
+Follow the “Getting Started” link to set up your application with App Center SDK. Then click on the service link for each App Center service that you want to enable in your app.
+
+-----Getting Started-----
+Xamarin: https://docs.microsoft.com/en-us/appcenter/sdk/getting-started/xamarin
```
Comment (guperrot) [inline nuget/readme.txt]: blank lines after the header should be better

File: nuget/readme.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+-------------------------------------------
+App Center SDK for .NET Readme
+-------------------------------------------
+
+Follow the “Getting Started” link to set up your application with App Center SDK. Then click on the service link for each App Center service that you want to enable in your app.
+
+-----Getting Started-----
```
Comment (guperrot) [inline nuget/readme.txt]: should have same length of ---- for all headers

File: nuget/readme.txt (hunk 1 above)
Comment (elamalani) [reply nuget/readme.txt]: Not sure what should be changed here?

Comment (guperrot) [reply nuget/readme.txt]: blank lines after each header section

**Comment by guperrot** [comment]
@elamalani edited myself

**Pull request merged**